internal/testutil/pkgdata.go

```diff
@@ -197,3 +197,16 @@ func Lnk(mode int64, path, target string) TarEntry {
 		},
 	}
 }
+
+// Hlk is a shortcut for creating a hard link TarEntry structure (with
+// tar.Typeflag set to tar.TypeLink). Hlk stands for "Hard LinK".
+func Hlk(mode int64, path, target string) TarEntry {
```

Comment:
I suggest _Hrd_ instead, as it makes it easier to read it into its meaning than "Hlk" (Hulk!? :smile:).

Reply:
Done in https://github.com/canonical/chisel/pull/179/commits/16adea9d0a032f643fed5860d60b1df359225bf9.